src/assets/scripts/client/tutorial/TutorialView.js

```diff
@@ -205,6 +205,15 @@ export default class TutorialView {
             position: tutorial_position
         });
 
+	    this.tutorial_step({
```

Comment:
someday I would love to re-do this entire tutorial.  makes me cringe a little.  but, alas, that is a very low priority.

Reply:
^ how I feel about a myriad of things in life...